data/abilities.js

```diff
@@ -1089,6 +1093,7 @@ exports.BattleAbilities = {
 		onResidual: function (pokemon) {
 			if (pokemon.status && this.isWeather(['raindance', 'primordialsea'])) {
 				this.debug('hydration');
+				this.add('-activate', pokemon, 'ability: Hydration');
```

Comment:
The protocol document says `-ability` is all ability activations that aren't on a `[from]` tag on another message. (I would know, I wrote it myself.)

`-activate` stands for "we couldn't think of a better message to put here that isn't super specific" more or less